contractcourt/channel_arbitrator.go

```diff
@@ -682,7 +682,7 @@ func (c *ChannelArbitrator) relaunchResolvers(commitSet *CommitSet,
 	// chain actions may exclude some information, but we cannot recover it
 	// for these older nodes at the moment.
 	var confirmedHTLCs []channeldb.HTLC
-	if commitSet != nil {
+	if commitSet != nil && commitSet.ConfCommitKey != nil {
```

Comment:
nice - curious tho, have you run into a panic before?

Reply:
No but I ran into the one in `constructChainActions` which was triggered by `TestChannelArbitratorPersistence` which writes an empty set to disc which will cause  `commitSet.ConfCommitKey` to be `nil`